app.py

```diff
@@ -9,7 +9,14 @@
 from apscheduler.triggers.cron import CronTrigger
 from flask import Flask
 
-from githubapp import GitHubApp, DirectoryClient, CRON_INTERVAL, TEST_MODE, ADD_MEMBER
+from githubapp import (
+    GitHubApp,
+    DirectoryClient,
+    CRON_INTERVAL,
+    TEST_MODE,
+    ADD_MEMBER,
+    COMPARE_ATTRIBUTE,
```

Comment:
@Chocrates you have the examples marked as `USER_ATTRIBUTE`, but here it is `COMPARE_ATTRIBUTE`... should we call this `USER_SYNC_ATTRIBUTE` to make it a bit more clear in the name what it is?